meta-networking/recipes-support/bmon/bmon_2.1.0.bb

```diff
@@ -0,0 +1,13 @@
+DESCRIPTION = "Portable Bandwidth Monitor and rate estimator"
+HOMEPAGE = "http://people.suug.ch/~tgr/bmon/"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+# only works with libnl-0.5.0
+DEPENDS = "libnl libconfuse ncurses"
+
+#SRCREV = "1b3f11bde315e221474f7d066ce4efb4ff4d39e3"
+SRC_URI = "git://github.com/tgraf/bmon.git;branch=master;tag=v4.0"
```

Comment:
Please do not use tags, lets use SRCREV like you have on commented line above, using tags means we need network to build always.